ota-plus-web/app/reactapp/style/ota-default/pages/_page-dashboard.scss

```diff
@@ -63,26 +63,22 @@
     justify-content: space-between;
     min-height: calc(100% - 50px);
     height: calc(100% - 50px);
-    max-height: 235px;
     overflow-y: hidden;
   }
 
```

Comment:
`Dashboard` page seems a little off. Possible solution: (replace `&__body` `&__list` `&__list-item`)
```suggestion
   &__body {
     display: flex;
     flex-direction: column;
     justify-content: space-between;
     min-height: calc(100% - 50px);
     height: calc(100% - 50px);
     overflow-y: scroll;
   }
 
   &__list {
     height: calc(100% / 3 - 10px / 3); 
     &:not(:last-child) {
       margin-bottom: 5px;
       @media all and (min-height:769px) {
         margin-bottom: 5px;
       }
     }
   }
 
   &__list-item {
     display: flex;
     height: 25%;
     min-height: 40px;
     padding: 0 20px;
     background-color: #fff;
     justify-content: space-between;
     cursor: pointer;
 
     &:nth-of-type(even) {
       background-color: $device-item-bg;
     }
   }
```

Reply:
It will stay as the way it is now. The design has been accepted by Benedicte.